### DISCUSSION TITLE
Curing Training Data

### DISCUSSION THREAD

**Discussion by jeffrey-cochran** [ultralytics/yolov5#8751] [Q&A] [answered]
There are some missing bounding boxes/labels in my data. I'd like to run the model on my training and validation data to identify the images most likely to contain errors. 

To do this, I just wanted to start with a list of images containing "incorrect" predictions. Is there flag for this somewhere, or do I need to make my own modifications?

**Comment by glenn-jocher** [comment, marked as answer]
@jeffrey-cochran val.py computed metrics per IoU threshold and per class but not per image (yet).

**Reply by jeffrey-cochran** [reply]
Thanks. I can implement something in the meantime
